l1-contracts/contracts/state-transition/chain-deps/facets/Admin.sol

```diff
@@ -228,16 +228,16 @@ contract AdminFacet is ZkSyncHyperchainBase, IAdmin {
     function forwardedBridgeBurn(
         address _settlementLayer,
         address _prevMsgSender,
-        bytes calldata
+        bytes calldata _data
     ) external payable override onlyBridgehub returns (bytes memory chainBridgeMintData) {
         require(s.settlementLayer == address(0), "Af: already migrated");
         require(_prevMsgSender == s.admin, "Af: not chainAdmin");
-        IStateTransitionManager stm = IStateTransitionManager(s.stateTransitionManager);
+
+        HyperchainCommitment memory chainCommitment = abi.decode(_data, (HyperchainCommitment));
```

Comment:
I don't think we need to carry the entire `HyperchainCommitment` to just pass one protocol version field